src/components/Navbar/index.spec.tsx

```diff
@@ -36,6 +38,14 @@ describe(__filename, () => {
     return store;
   };
 
+  it('renders a brand link', () => {
```

Comment:
I don't see anything in this test that will be helpful so you could remove it. Otherwise it will just be one extra thing you'll have to update when the JSX changes. 

All the test does is duplicate the static configuration of `<Navbar.Brand>` and `<Link>`. Static configuration doesn't changed based on any inputs so it has no potential for breakage. Also, the test is not executing any code that isn't already executed so it's not providing additional coverage. Finally, none of the JSX tested gets rendered conditionally so asserting its presence is not helpful.

Reply:
That's right.